sdk/purview/Azure.Analytics.Purview.DataMap/src/Generated/Docs/Entity.xml

```diff
@@ -62,16 +62,22 @@ AtlasEntityWithExtInfo body = new AtlasEntityWithExtInfo
         Version = 0L,
         Contacts =
         {
-            ["Expert"] = {new ContactInfo
```

Comment:
so in this case, we have:
```
public class Foo
{
	public IDictionary<string, IList<ContactInfo>> Contacts {get;}
}
```
if we use it in this way:
```
var foo = new Foo()
{
	Contacts = {
		["Expert"]  = {
			new ContactInfo()
		}
	}
}
```
this is actually grammar sugar for this code:
```
var foo = new Foo();
foo.Contacts["Expert"].Add(new ContactInfo());
```
our generated code would initialize `Contacts` with an empty dictionary, therefore the first part is fine. but the second part `.Add` will have a NullReferenceException because we never give an instance for the given key.

This code compiles fine but it will never work.